### PR TITLE
(Upgrade): Update the esm-patient-common-lib version.

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes the ticket number in the format `OHRI-123 My PR title`.
- [ ] My work includes tests or is validated by existing tests.

## Summary
- This PR updates the `openmrs` and `common-lib` versions, this necessitated some changes since some workspace functions were moved to other places or are no longer being exported. But this fixes the workspace bugs we have been seeing.
- This PR also updates the `openmrs` and common lib versions inside the `ohri-esm-commons-lib`.

## Screenshots

Uploading Workspaces-fix.mov…

## Related Issue
- No ticket

### CODE REVIEW

**Comment by denniskigen** [comment]
@hadijahkyampeire please run `yarn` again and commit the updated lockfile.

File: packages/esm-commons-lib/src/index.ts [hunk 1]
```diff
@@ -49,3 +52,12 @@ export * from './components/expandable-list-widget/expandable-list.component';
...
 export * from './components/patient-table/patient-table.component';
 export * from './hooks/useLastEncounter';
+
+// Workspace registration moved to the index.ts and routes.json
+const options = {
+  featureName: 'ohri-forms-workspace-item',
+  moduleName: "@ohri/openmrs-esm-ohri-commons-lib",
+};
+
+// t('ohriForms', "Ohri Forms")
```
Comment (denniskigen) [inline packages/esm-commons-lib/src/index.ts:62]:
```suggestion
// t('ohriForms', "OHRI Forms")
```

Seems more correct to me.

**Review by alaboso — APPROVED** [review]

File: packages/esm-commons-lib/package.json [hunk 2]
```diff
@@ -39,8 +39,8 @@
     "yup": "^0.29.1"
   },
   "peerDependencies": {
-    "@openmrs/esm-framework": "5.x",
-    "@openmrs/esm-patient-common-lib": "6.x",
+    "@openmrs/esm-framework": "*",
+    "@openmrs/esm-patient-common-lib": "*",
```
Comment (denniskigen) [inline packages/esm-commons-lib/package.json:43]: These should be X.x versions, right @ibacher?

Comment (ibacher) [reply packages/esm-commons-lib/package.json:43]: Yes, although I don't know that that's a hard requirement.

**Review by denniskigen — APPROVED** [review]

**Review by eudson — APPROVED** [review]

**Pull request merged**